PROCESSORS.md

```diff
@@ -389,6 +390,40 @@ In the list below, the names of required properties appear in bold. Any other pr
 |success|Successfully retrieved OPC-UA nodes|
 
 
+## FetchS3Object
+
+### Description
+
+Retrieves the contents of an S3 Object and writes it to the content of a FlowFile
+### Properties
+
+In the list below, the names of required properties appear in bold. Any other properties (not in bold) are considered optional. The table also indicates any default values, and whether a property supports the NiFi Expression Language.
+
+| Name | Default Value | Allowable Values | Description |
+| - | - | - | - |
+|Object Key|||The key of the S3 object. If none is given the filename attribute will be used by default.<br/>**Supports Expression Language: true**|
+|**Bucket**|||The S3 bucket<br/>**Supports Expression Language: true**|
+|Access Key|||AWS account access key<br/>**Supports Expression Language: true**|
+|Secret Key|||AWS account secret key<br/>**Supports Expression Language: true**|
+|Credentials File|||Path to a file containing AWS access key and secret key in properties file format. Properties used: accessKey and secretKey|
+|AWS Credentials Provider service|||The name of the AWS Credentials Provider controller service that is used to obtain AWS credentials.|
+|**Region**|us-west-2|af-south-1<br/>ap-east-1<br/>ap-northeast-1<br/>ap-northeast-2<br/>ap-northeast-3<br/>ap-south-1<br/>ap-southeast-1<br/>ap-southeast-2<br/>ca-central-1<br/>cn-north-1<br/>cn-northwest-1<br/>eu-central-1<br/>eu-north-1<br/>eu-south-1<br/>eu-west-1<br/>eu-west-2<br/>eu-west-3<br/>me-south-1<br/>sa-east-1<br/>us-east-1<br/>us-east-2<br/>us-gov-east-1<br/>us-gov-west-1<br/>us-west-1<br/>us-west-2|AWS Region|
```

Comment:
Very minor, but this list of allowable values does not look well at all when rendered:
![image](https://user-images.githubusercontent.com/64011968/103784353-242a2880-503a-11eb-80d0-da77c711ab14.png)

It might be worth doing some manual tweaking on it, eg changing font size and adding some padding to the top and bottom. Same stands for DeleteS3Object.

Reply:
I adjusted the cell width with a fix but scalable div in [9417f98](https://github.com/apache/nifi-minifi-cpp/pull/970/commits/9417f98d7314b35bf2d346fbd83aaa92ff58b58d). The region values should not wrap anymore.